install_config/syncing_groups_with_ldap.adoc

```diff
@@ -433,6 +437,155 @@ users:
 <1> The name of the Group as specified by the user-defined name mapping.
 ====
 
+[[rfc2307-with-error-tolerances]]
+=== RFC 2307 with User-Defined Error Tolerances
+
+By default, if groups being synced contain members whose entries that are outside of the
+scope defined in the member query, the group sync fails with an error like:
+
+----
+Error determining LDAP group membership for "<group>": membership lookup for user "<user>" in group "<group>" failed because of "search for entry with dn="<user-dn>" would search outside of the base dn specified (dn="<base-dn>")".
+----
+
+Most often, failures of this type indicated a mis-configured user query. However, if it
```

Comment:
This often indicates a mis-configured `baseDN` in the `usersQuery`. However, in cases where the `baseDN` intentionally does not contain some of the members of the group, setting `tolerateMemberOutOfScopeErrors: true` will allow the group sync to continue.